recipes/basedpyright/meta.yaml

```diff
@@ -24,12 +22,14 @@ requirements:
   host:
     - python {{ python_min }}
     - pdm-backend >=2.3.0
+    - nodejs-wheel >=20.13.1
     - typing_extensions >=4.12.2
     - docify >=1.0.0
     - pip
     - nodejs >=20
```

Comment:
The nodejs-wheel package should bring this to you. 

```suggestion
```

Reply:
Oops, `git pull --rebase` failure on my part